module-bsp/board/rt1051/drivers/RT1051DriverPWM.cpp

```diff
@@ -171,18 +171,19 @@ namespace drivers
         base->SM[pwmModule].CTRL2 |= PWM_CTRL2_FORCE(1U);
     }
 
-    void RT1051DriverPWM::UpdateClockFrequency(std::uint32_t newFrequency)
+    void RT1051DriverPWM::UpdateClockFrequency(bsp::CpuFrequencyMHz newFrequency)
     {
         cpp_freertos::LockGuard lock(frequencyChangeMutex);
+        std::uint32_t convertedFrequency = static_cast<std::uint32_t>(newFrequency) * 1000000;
```

Comment:
Please add at least constexpr for it like `constexpr auto mhzToHz = 1000000` to describe what is done here